gradle/versions.gradle

```
@@ -19,7 +19,7 @@
         junitVersion = "4.13.2"
         restAssuredVersion = "4.5.1"
         explorerApiCommonVersion = "2.0.38"
```

Once the explorer is released, please update this value.
```suggestion
        explorerApiCommonVersion = "2.0.39"
```